### PR TITLE
Adding next steps, passwordless warnings, linting

### PR DESCRIPTION
**Pull request by jeffreylees**

### CODE REVIEW

File: articles/libraries/lock-android/index.md [hunk 1]
```diff
@@ -257,18 +257,15 @@ For a full list of Lock's configuration options, check out the [Lock for Android
...
-    <% } %>
-    </span>
-  </li>
-  <% } %>
-<% }); %>
-</ul>
+## Next Steps
+
+::: next-steps
+
```
Comment (mpaktiti) [inline articles/libraries/lock-android/index.md]: The max, per guidelines, is 5 items. Can you remove one? Logout looks like a good candidate. Also the whitespace is note required inside the `next-steps` element

File: articles/libraries/lock-android/passwordless.md [hunk 2]
```diff
@@ -6,7 +6,9 @@ description: Guide on implementing Passwordless authentication with Lock for And
 # Lock Passwordless
 
 ::: warning
-This feature is disabled by default for new tenants as of 8 June 2017. If you would like this feature enabled, please contact support to discuss your use case and prevent the possibility of introducing security vulnerabilities. Please see [Client Grant Types](/clients/client-grant-types) for more information.
+Passwordless on native platforms is disabled by default for new tenants as of 8 June 2017. If you would like this feature enabled, please contact support to discuss your use case and prevent the possibility of introducing security vulnerabilities. Please see [Client Grant Types](/clients/client-grant-types) for more information.
```
Comment (mpaktiti) [inline articles/libraries/lock-android/passwordless.md]: Too long, warnings and notes should be only for brief content (one to four lines). Replace with `panel-warning`.

Comment (mpaktiti) [reply articles/libraries/lock-android/passwordless.md]: Also the `check into` does not seem right, it's like checking into a hotel or sth.

Comment (mpaktiti) [reply articles/libraries/lock-android/passwordless.md]: Try to keep the text short in panels (ie `As an alternative to using native passwordless,` -> `Alternatively,`

File: articles/libraries/lock-ios/v2/index.md [hunk 3]
```diff
@@ -95,7 +95,9 @@ Lock
 ## Implementation of Lock Passwordless
 
 ::: warning
-This feature is disabled by default for new tenants as of 8 June 2017. If you would like this feature enabled, please contact support to discuss your use case and prevent the possibility of introducing security vulnerabilities. Please see [Client Grant Types](/clients/client-grant-types) for more information.
+Passwordless on native platforms is disabled by default for new tenants as of 8 June 2017. If you would like this feature enabled, please contact support to discuss your use case and prevent the possibility of introducing security vulnerabilities. Please see [Client Grant Types](/clients/client-grant-types) for more information.
```
Comment (mpaktiti) [inline articles/libraries/lock-ios/v2/index.md]: Same comments apply as in `articles/libraries/lock-android/passwordless.md`

File: articles/libraries/lock-ios/v2/index.md [hunk 4]
```diff
@@ -230,10 +232,15 @@ Lock
...
+## Next Steps
+
+::: next-steps
 
-- [Styles Customization](/libraries/lock-ios/v2/customization) - customize the look and feel of Lock
-- [Behavior Configuration](/libraries/lock-ios/v2/configuration) - configure the behavior of Lock
-- [Custom Fields](/libraries/lock-ios/v2/custom-fields) - adding custom signup fields to Lock
-- [Internationalization](/libraries/lock-ios/v2/internationalization) - internationalization and localization support in Lock
-- [Migration Guide for v2](/libraries/lock-ios/v2/migration) - migrate from Lock v1 to Lock v2
+- [Customizing the Style of Lock](/libraries/lock-ios/v2/customization)
```
Comment (mpaktiti) [inline articles/libraries/lock-ios/v2/index.md]: Same comment as above, ,leave max 5 items here

**Review by mpaktiti — CHANGES_REQUESTED** [review]
Some fixes on UI elements

File: articles/libraries/lock-android/index.md (hunk 1 above)
Comment (jeffreylees) [reply articles/libraries/lock-android/index.md]: Does it matter that the linter has issues with the spacing around lists, or will that not come up here?

Adding Logout reference to these docs was part of the reason I was implementing the next-steps widget altogether 🙊  - maybe I can put that somewhere else though... will look at it.

**Pull request merged**